Makefile.main

```diff
@@ -252,6 +261,20 @@ ci-script: | test-coverage codecov packages
 	@echo
 endif
 
+install-helm:
+	GET_HELM=`mktemp` ; \
+	trap "rm -f $${GET_HELM}" EXIT; \
+	curl -L https://raw.githubusercontent.com/helm/helm/master/scripts/get > $${GET_HELM}; \
+	chmod +x $${GET_HELM} ; \
+	DESIRED_VERSION=$(HELM_VERSION) $${GET_HELM}
+
+helm-deploy: install-helm
```

Comment:
I rather keep, the name agnostic to method being used, so `deploy` sound better.

Reply:
fixed!